tools/props/Versions.props

```diff
@@ -35,6 +35,7 @@
     <MicrosoftIdentityModelJsonWebTokensVersion>7.5.0</MicrosoftIdentityModelJsonWebTokensVersion>
     <MicrosoftIdentityModelProtocolsOpenIdConnectVersion>7.5.0</MicrosoftIdentityModelProtocolsOpenIdConnectVersion>
     <SystemTextJsonVersion>6.0.10</SystemTextJsonVersion>
```

Comment:
This version of System.Text.Json is part of .NET 6.0. Same point with `SystemTextEncodingsWebVersion` on line 29.

Reply:
They are both not referenced in the netcore project.
I think the variable here should be moved up to the "netfx dependencies" section